Gemfile.lock

```diff
@@ -378,15 +381,18 @@ GEM
     ffi-compiler (1.0.1)
       ffi (>= 1.0.0)
       rake
-    font-awesome-rails (4.7.0.8)
-      railties (>= 3.2, < 8.0)
+    font-awesome-sass (5.15.1)
+      sassc (>= 1.11)
```

Comment:
Oh maybe this is the reason why we still need `sassc`... 
It's probably would be interesting to try and use font-awesome that it baked into PF4 (in `node_modules`), so we can get rid of this dependency?
Not sure it's feasible though, maybe @josemigallas would know...